packages/itwinui-react/src/core/DatePicker/DatePicker.tsx

```diff
@@ -190,6 +190,30 @@ type DatePickerProps = {
    * @default false
    */
   showYearSelection?: boolean;
+  /**
+   * Allows props to be passed for calendar month year.
+   */
+  calendarMonthYearProps?: React.ComponentProps<'div'>;
```

Comment:
reopening the discussion about naming. I think the `calendar` prefix can be removed from all of these props (except of course `calendarProps`). it just makes the name longer for no benefit.